train.py

```diff
@@ -87,6 +92,7 @@ def main(job_config: JobConfig):
 
     model_name = job_config.model.name
     world_mesh = parallel_dims.build_mesh(device_type="cuda")
+    init_device = "cpu" if job_config.checkpoint.create_seed_checkpoint else "cuda"
```

Comment:
What is this doing?

Reply:
`init_device` refers to the device used for each rank

Reply:
by design